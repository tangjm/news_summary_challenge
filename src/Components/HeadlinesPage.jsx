import ArticleBlock from './ArticleBlock';

const HeadlinesPage = () => {
	return (
		<>
			<ArticleBlock />
			<ArticleBlock />
			<ArticleBlock />
		</>
	)
}

export default HeadlinesPage;
