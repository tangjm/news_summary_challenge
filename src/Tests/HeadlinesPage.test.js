import { render } from '@testing-library/react';
import HeadlinesPage from '../Components/HeadlinesPage';

test(`Article Block matches snapshot`, () => {
	const headlinesPage = render(<HeadlinesPage />);
	expect(headlinesPage).toMatchSnapshot();
})