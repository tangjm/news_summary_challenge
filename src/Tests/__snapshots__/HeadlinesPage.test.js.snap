// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article Block matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <img
          alt="article thumnail"
          src="https://cdn-media-1.freecodecamp.org/images/1*y6C4nSvy2Woe0m7bWEn4BA.png"
        />
        <h2>
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum, quas.
        </h2>
      </div>
      <div>
        <img
          alt="article thumnail"
          src="https://cdn-media-1.freecodecamp.org/images/1*y6C4nSvy2Woe0m7bWEn4BA.png"
        />
        <h2>
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum, quas.
        </h2>
      </div>
      <div>
        <img
          alt="article thumnail"
          src="https://cdn-media-1.freecodecamp.org/images/1*y6C4nSvy2Woe0m7bWEn4BA.png"
        />
        <h2>
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum, quas.
        </h2>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <img
        alt="article thumnail"
        src="https://cdn-media-1.freecodecamp.org/images/1*y6C4nSvy2Woe0m7bWEn4BA.png"
      />
      <h2>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum, quas.
      </h2>
    </div>
    <div>
      <img
        alt="article thumnail"
        src="https://cdn-media-1.freecodecamp.org/images/1*y6C4nSvy2Woe0m7bWEn4BA.png"
      />
      <h2>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum, quas.
      </h2>
    </div>
    <div>
      <img
        alt="article thumnail"
        src="https://cdn-media-1.freecodecamp.org/images/1*y6C4nSvy2Woe0m7bWEn4BA.png"
      />
      <h2>
        Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum, quas.
      </h2>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
